### PR TITLE
Add common files for ONTAP and SOLIDFIRE unit tests

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Add common files for ONTAP and SOLIDFIRE unit tests, and documentation changes for python 3

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below
ansible 2.8.0.dev0 (netapp-unittest2 5f3e5053c0) last updated 2018/11/15 10:24:07 (GMT -700)
  config file = None
  configured module search path = [u'/Users/chrisarchibald/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/chrisarchibald/Code/github/ansible/lib/ansible
  executable location = /Users/chrisarchibald/Code/github/ansible/bin/ansible
  python version = 2.7.15 (default, Oct  2 2018, 12:50:38) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
```

### CODE REVIEW

File: test/units/modules/storage/netapp/test_netapp.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2018 NetApp
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+''' unit tests for module_utils netapp.py '''
+from __future__ import absolute_import, division, print_function
+
+try:
+    from ansible.module_utils.ansible_release import __version__ as ansible_version
+except ImportError:
+    ansible_version = 'unknown'
```
Comment (mattclay) [inline test/units/modules/storage/netapp/test_netapp.py]: A try/except block shouldn't be needed to import the ansible version.

File: lib/ansible/module_utils/netapp.py [hunk 2]
```diff
@@ -40,17 +38,13 @@
...
-HAS_NETAPP_LIB = False
 try:
     from netapp_lib.api.zapi import zapi
-    from netapp_lib.api.zapi import errors as zapi_errors
     HAS_NETAPP_LIB = True
-except:
+except ImportError:
     HAS_NETAPP_LIB = False
 
+import ssl
```
Comment (lonico) [inline lib/ansible/module_utils/netapp.py:47]: we should move this to the top, after import os.
But we can do it later.

**Review by lonico — APPROVED** [review]

**Review by schmots1 — APPROVED** [review]
shipit

**Comment by gundalow** [comment]
@carchi8py thank you for the PR
@lonico @schmots1 Thank you for the review.

Merged into `devel` for release in Ansible 2.8

**Pull request merged**